### PR TITLE
DM-39662 Add ApPipeWithFakes.yaml to LsstCamImSim

### PR DESCRIPTION
**Pull request by BrunoSanchez**
Adding the yaml file for ImSim ApPipeWithFakes which was missing.

Requesting help to test with DC2 data at usdf.

### CODE REVIEW

File: pipelines/LsstCamImSim/ApPipeWithFakes.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+tasks:
+  processVisitFakes:
+    class: lsst.pipe.tasks.processCcdWithFakes.ProcessCcdWithVariableFakesTask
+    config:
+      file: $AP_PIPE_DIR/config/HSC/processCcdWithFakes.py
+      connections.coaddName: parameters.CoaddName
+      insertFakes.doSubSelectSources: True
+      insertFakes.select_col: 'isVisitSource'
+      calibrate.photoCal.match.referenceSelection.magLimit.fluxField: i_flux
+      calibrate.photoCal.match.referenceSelection.magLimit.maximum: 22.0
```
Comment (isullivan) [inline pipelines/LsstCamImSim/ApPipeWithFakes.yaml:30]: Remove `processVisitFakes` and its configs. This was needed for HSC, and not for other instruments (note the `HSC` in the file path).

**Review by isullivan — APPROVED** [review]
See the note about removing `processVisitFakes`

**Comment by BrunoSanchez** [comment]
Check that changes are OK now.

**Comment by BrunoSanchez** [comment]
After CI successful using imsim images I think this can be merged. Still would like confirmation from reviewers.
Jenkins run is linked here:
https://rubin-ci.slac.stanford.edu/blue/organizations/jenkins/stack-os-matrix/detail/stack-os-matrix/285/pipeline

**Pull request merged**